app/build.gradle

```diff
@@ -70,6 +70,7 @@ dependencies {
     compile "com.android.support:cardview-v7:$rootProject.supportLibraryVersion"
     compile "com.google.android.gms:play-services-maps:$rootProject.playServicesVersion"
     compile "com.android.support:recyclerview-v7:$rootProject.supportLibraryVersion"
+    compile "com.android.support:gridlayout-v7:$rootProject.supportLibraryVersion"
```

Comment:
why do you need `support:gridlayout`, I mean you can do same thing using `support:recyclerview`
` GridLayoutManager mLayoutManager = new GridLayoutManager(getActivity(), grid_count);`